Dockerfile

```diff
@@ -1,6 +1,6 @@
 
 FROM centos:centos6
-RUN curl -sL https://rpm.nodesource.com/setup_8.x | bash -
+RUN curl -sL https://rpm.nodesource.com/setup_12.x | bash -
```

Comment:
While we are modifying this, why not just use the official node images instead of doing this work ourselves? https://hub.docker.com/_/node/